csharp/extractor/Semmle.Extraction.CSharp/Extractor/Analyser.cs

```diff
@@ -346,7 +346,7 @@ public virtual void Dispose()
         /// </summary>
         public void LogExtractorInfo()
         {
-            Logger.LogInfo($"  Extractor: {Environment.GetCommandLineArgs().First()}");
+            Logger.LogInfo($"  Extractor: {Environment.GetCommandLineArgs()[0]}");
```

Comment:
Why do you prefer `[0]` over `.First()`? I suppose it makes sense for performance critical loops, but are there other reasons as well?

Reply:
No other reason than perf. I changed a couple of `array.First()` calls to `array[0]`.